lib/cassandra.rb

```diff
@@ -117,10 +122,14 @@ module Cassandra
   #   found by the default Token-Aware Load Balancing Policy should be
   #   shuffled. See {Cassandra::LoadBalancing::Policies::TokenAware#initialize Token-Aware Load Balancing Policy}.
   #
+  # @option options [Hash<String|Symbol, ExecutionProfile>] :execution_profiles (nil)
+  #   Hash of {Cassandra::Execution::Profile}s that are available for client use (e.g.
+  #   {Session#execute}, {Session#execute_async}, {Session#prepare}, and {Session#prepare_async}).
+  #
   # @option options [Numeric] :connect_timeout (10) connection timeout in
   #   seconds. Setting value to `nil` will reset it to 5 seconds.
   #
-  # @option options [Numeric] :timeout (10) request execution timeout in
+  # @option options [Numeric] :timeout (12) request execution timeout in
```

Comment:
is this just a missing doc update or an actual default change? You might also want to update the CHANGELOG in this PR.